src/style.css

```
@@ -1 +1,75 @@
 /* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.nav__header {
+  display: flex;
+  padding: 0 40px;
+  height: 60px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  padding-left: 10px;
+}
+
+.nav {
+  padding: 0;
+  margin: 0;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  height: 60px;
+  list-style: none;
+}
+
+.nav__item {
+  position: relative;
+  margin: 0 10px;
+  cursor: pointer;
+  color: #000;
+}
+
+.nav__link {
+  padding: 23px 0;
+  display: inline-block;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: inherit;
+}
+
+.nav__item:hover {
+  color: #00acdc;
+}
+
+.nav__item:active {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.nav__item:active > .nav__link::after,
+.is-active::after {
+  content: "";
```

This line should display only for elements with class is-active
```suggestion
.is-active::after {
  content: "";
```